exports.go

```diff
@@ -124,6 +126,28 @@ func (e *Export) Validate(vr *ValidationResults) {
 		e.Latency.Validate(vr)
 	}
 	e.Subject.Validate(vr)
+
+	if e.AccountTokenPosition > 0 {
+		if !e.Subject.HasWildCards() {
+			vr.AddError("Account Token Position can only be used when wildcard subjects: %s", e.Subject)
```

Comment:
Account token position can onlyh be used 'with' wildcarded subjects?

Reply:
yes, seems pointless without. wouldn't you agree?